src/sage/rings/function_field/function_field_polymod.py

```diff
@@ -1771,17 +1786,18 @@ def exact_constant_field(self, name='t'):
 
         EXAMPLES::
 
-            sage: K.<x> = FunctionField(GF(3)); _.<Y> = K[]                             # optional - sage.rings.finite_rings
-            sage: f = Y^2 - x*Y + x^2 + 1 # irreducible but not absolutely irreducible  # optional - sage.rings.finite_rings
-            sage: L.<y> = K.extension(f)                                                # optional - sage.rings.finite_rings
-            sage: L.genus()                                                             # optional - sage.rings.finite_rings
+            sage: # needs sage.rings.finite_rings
```

Comment:
`sage.rings.finite_rings` is for almost all lines of examples / test that It could be at the top of the file.

Reply:
Actually because prime finite fields are always available now, I was able to remove most of these tags.